src/modules/feedback/components/FeedbackBox.js

```diff
@@ -46,6 +48,11 @@ type PropsType = {|
  * Renders all necessary inputs for a Feedback and posts the data to the feedback endpoint
  */
 export class FeedbackBox extends React.PureComponent<PropsType> {
+  handleClick = () => {
```

Comment:
No need to handle this here. This should be handled in the top component (`FeedbackModal` in this case).